lighthouse-core/test/fraggle-rock/scenarios/api-test-pptr.js

```diff
@@ -141,7 +141,7 @@ describe('Fraggle Rock API', () => {
 
     it('should compute both snapshot & timespan results', async () => {
       const {page, serverBaseUrl} = state;
-      const result = await lighthouse.navigation({page, url: `${serverBaseUrl}/index.html`});
+      const result = await lighthouse.navigation({page, requestor: `${serverBaseUrl}/index.html`});
```

Comment:
okay API-wise this feels weird.

here we have 1 big object `fr.navigation({page: Puppteer.page, requestor: LH.NavigationRequestor})`

but in a flow we have requestor then 1 big object `flow.navigate(LH.NavigationRequestor, Options?}`
To be fair, in a flow we already received the page, so it's not apples-to-apples but… 1) the API difference feels weird and 2) much more important to me…  this makes `requestor` a user-exposed term. I didn't expect that and I really don't like it.

i'd definitely prefer 

```js
fr.navigation(LH.NavigationRequestor, Options)
```

yes, this means there's now an inconsistency in the method signatures for navigation vs snapshot/startTimespan.. but the requestor argument is such a critical piece it deserves first param spot, and it also means that we don't need to expose the term in our public API.

Reply:
Yeah this sounds good.

Imo we should go ahead and make `page` it's own parameter as well for all three functions but that can be a future PR/discussion.